### PR TITLE
#2591 Cypress: Update tests to use new fixture approach 

### PR DESCRIPTION
**Pull request by cristinastefan90**
https://github.com/metasfresh/me03/issues/2591

### CODE REVIEW

File: cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js [hunk 1]
```diff
@@ -29,21 +29,49 @@ import { getLanguageSpecific } from '../../support/utils/utils';
...
-  const productName = 'Convenience Salat 250g';
-  const originalQuantity = 200;
-  const shipmentNotificationInboxText = `Lieferung [number] für Partner ${businessPartnerName} wurde erstellt.`;
-  const shipmentNotificationModalText = 'Created: 1 WorkPackage Queue;';
-  const shipmentQuantityTypeOption = 'Quantity to deliver';
-  const generateInvoicesNotificationModalText =
-    'Fakturlauf mit 1 Rechnungen eingeplant. Es sind bereits 0 zu erstellende Rechnungen in der Warteschlange, die vorher verarbeitet werden.';
-
-  const salesInvoiceDocumentType = 'Sales Invoice';
+  // const businessPartnerName = 'Test Lieferant 1';
```
Comment (TheBestPessimist) [inline cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js]: Can the comments here be deleted please?

File: cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js [hunk 2]
```diff
@@ -29,21 +29,49 @@ import { getLanguageSpecific } from '../../support/utils/utils';
...
+  let shipmentNotificationModalText;
+  let shipmentQuantityTypeOption;
+  let generateInvoicesNotificationModalText;
+  let salesInvoiceDocumentType;
 
   // for test
   let salesOrderDocumentNumber;
   let salesOrderRecordId;
-  let totalAmountToPay = 0;
+  let totalAmountToPay;
```
Comment (TheBestPessimist) [inline cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js]: totalAmountToPay is a test variable and should not appear in the fixture

File: cypress/fixtures/price/change_product_prices_and_add_a_new_one.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (TheBestPessimist) [inline cypress/fixtures/price/change_product_prices_and_add_a_new_one.json]: I dont think the .js couterpart is added to this commit.

File: cypress/integration/dunning/create_dunning_candidates.js [hunk 4]
```diff
@@ -61,6 +66,7 @@ describe('Create Dunning Candidates', function() {
   it('Prepare customer bpartner (via api)', function() {
     cy.fixture('sales/simple_customer.json').then(customerJson => {
       const bpartner = new BPartner({ ...customerJson, name: businessPartnerName })
+        .setCustomer(true)
```
[the remark this answers is not in the document]
Comment (cristinastefan90) [reply cypress/integration/dunning/create_dunning_candidates.js]: yup, done.

File: cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js (hunk 1 above)
Comment (cristinastefan90) [reply cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js]: yes, done.

File: cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js (hunk 2 above)
Comment (cristinastefan90) [reply cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js]: done.

File: cypress/fixtures/price/change_product_prices_and_add_a_new_one.json (hunk 3 above)
Comment (cristinastefan90) [reply cypress/fixtures/price/change_product_prices_and_add_a_new_one.json]: no, as the test is not working currently, will be done with another branch/commit.

File: cypress/integration/payment/create_manual_payment.js [hunk 5]
```diff
@@ -5,35 +5,79 @@ import { SalesInvoice, SalesInvoiceLine } from '../../support/utils/sales_invoic
...
 import { Bank } from '../../support/utils/bank';
 import { Builder } from '../../support/utils/builder';
-import { getLanguageSpecific, humanReadableNow } from '../../support/utils/utils';
+import { getLanguageSpecific, appendHumanReadableNow } from '../../support/utils/utils';
 
 describe('Create a manual Payment for a Sales Invoice', function() {
-  const date = humanReadableNow();
-
-  const salesInvoiceTargetDocumentType = 'Sales Invoice';
+  // const salesInvoiceTargetDocumentType = 'Sales Invoice';
```
[the remark this answers is not in the document]
Comment (TheBestPessimist) [reply cypress/integration/payment/create_manual_payment.js]: Bump

[the remark this answers is not in the document]
Comment (cristinastefan90) [reply cypress/integration/payment/create_manual_payment.js]: done

**Pull request merged**